actors/power/src/lib.rs

```diff
@@ -298,7 +298,7 @@ impl Actor {
 
     fn submit_porep_for_bulk_verify(
         rt: &mut impl Runtime,
-        seal_info: SealVerifyInfo,
+        params: SubmitPoRepForBulkVerifyParams,
```

Comment:
Hmm, this should probably be a rename of SealVerifyInfo, but that's not in this repo so you can't. 👍 for this change now, but I think those types are defined in the wrong place.